tests/dummy/app/controllers/application.js

```diff
@@ -2,6 +2,7 @@ import Ember from 'ember';
 
 export default Ember.Controller.extend({
   initialDatetime: new Date(),
+  datetime: new Date('12/25/1994'),
```

Comment:
No, this is to show it sets the date specified while in inline mode;
otherwise, the default just sets to the current date which is why it passed
validation first go around.

On Wed, Nov 11, 2015 at 3:38 PM, Kelly Selden <notifications@github.com>
wrote:

> In tests/dummy/app/controllers/application.js
> <https://github.com/kellyselden/ember-datetimepicker/pull/1#discussion_r44582932>
> :
>
> > @@ -2,6 +2,7 @@ import Ember from 'ember';
> >
> >  export default Ember.Controller.extend({
> >    initialDatetime: new Date(),
> > +  datetime: new Date('12/25/1994'),
>
> was this accidentally left in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kellyselden/ember-datetimepicker/pull/1/files#r44582932>
> .
>



-- 
Paul Mead
*Owner/Founder*
*Cell*: 616-826-9882
*Phone*: 616-319-1173